WordPress/src/main/res/drawable/ic_shield_warning_white.xml

```diff
@@ -5,5 +5,5 @@
     android:viewportHeight="162">
   <path
       android:pathData="M69,8L12,33.046V70.614C12,105.364 36.32,137.861 69,145.75C101.68,137.861 126,105.364 126,70.614V33.046L69,8ZM62.667,45.568H75.333V83.136H62.667V45.568ZM62.667,95.659H75.333V108.182H62.667"
-      android:fillColor="#DC3232"/>
+      android:fillColor="#ffffff"/>
```

Comment:
❓: Should we replace `ffffff` with `@android:color/white` similar to [this one](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/drawable/bg_oval_white_24dp.xml#L9)?

Reply:
Good call, fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13824/commits/27af469234e5bd51b6a4ed73e3bed6959b0b59f7